app/controllers/bill-runs.controller.js

```diff
@@ -185,6 +186,13 @@ async function submitRemoveLicence (request, h) {
   return h.redirect(`/system/bill-runs/${billRunId}/review`)
 }
 
+async function submitReview (request, h) {
+  const { id } = request.params
+  await SubmitReviewBillRunService.go(request.payload, request.yar)
```

Comment:
NIT - add some whitespace

```suggestion

  await SubmitReviewBillRunService.go(request.payload, request.yar)
```